core/src/main/java/org/pentaho/di/core/plugins/KettleURLClassLoader.java

```diff
@@ -104,7 +104,7 @@ public String getName() {
   protected synchronized Class<?> loadClass( String arg0, boolean arg1 ) throws ClassNotFoundException {
     try {
       return loadClassFromThisLoader( arg0, arg1 );
-    } catch ( ClassNotFoundException | NoClassDefFoundError e ) {
+    } catch ( ClassNotFoundException | NoClassDefFoundError | SecurityException e ) {
```

Comment:
Would prefer logging here - need to log the exception somewhere. Swallowing it is not a good fix.